card/card.gemspec

```diff
@@ -62,7 +62,7 @@ Gem::Specification.new do |s|
 
     # other
     ["diff-lcs",                   "~> 1.3"], # content diffs in histories
-    ["recaptcha",                  "~> 4.3"],
+    ["recaptcha",                  "~> 4.13.1"],
```

Comment:
issues with the more recent versions?

Reply:
anything worth a code comment?

Reply:
more recent? it's the most recent version

Reply:
DOH.